src/test/kotlin/com/igorwojda/string/capitalizesentence/challenge.kt

```diff
@@ -4,7 +4,12 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun capitalizeSentence(str: String): String {
-    TODO("not implemented")
+    val strs=str.split(" ")
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html is a powerful tool

Reply:
return str.split("_").map { it.toLowerCase().capitalize() }.joinToString().replace(", ","").decapitalize()

Reply:
return str.split(" ").joinToString(" ") { it.capitalize() }